Marlin/ultralcd.cpp

```diff
@@ -2777,8 +2777,13 @@ void lcd_update() {
 
     if (LCD_HANDLER_CONDITION) {
 
-      if (lcdDrawUpdate) {
-
+      #if ENABLED(DOGLCD)
+        static uint8_t drawing_screen = 0;
+        if (lcdDrawUpdate || drawing_screen) {
+          if (!drawing_screen)
```

Comment:
Since this combination evaluates to this…
```cpp
if ((lcdDrawUpdate || drawing_screen) && !drawing_screen) { . . .
```
…I guess it can be simplified to this…
```cpp
if (lcdDrawUpdate && !drawing_screen) { . . .
```